app/Repositories/ContactRepositoryTimer.php

```diff
@@ -0,0 +1,33 @@
+<?php namespace App\Repositories;
+
+use App\Models\Contact;
+use Illuminate\Contracts\Pagination\LengthAwarePaginator;
+
+class ContactRepositoryTimer implements ContactRepository
```

Comment:
A timer for the repo. This is nice and simple, it times how long it took to get the list of contacts.